### DISCUSSION TITLE
Access/Export Podcast history

### DISCUSSION THREAD

**Discussion by Julian4711** [thrillfall/nextcloud-gpodder#171] [Q&A]
Hi,

I am wondering if it is possible to export data about listened episodes, timestamps and so on from the gpodder database?
Where is it actually located in a nextcloud setup?

Thank you!

**Comment by JonOfUs** [comment]
Hi, the data is located in the main Nextcloud database, in the tables `gpodder_subscriptions` and `gpodder_episode_action` (mostly with `oc_` prefix). Be careful not to delete anything in the db if you access it, best to only access it as read-only.
But the data is possibly not easy to read and to re-import into other systems.
